src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -71,6 +71,28 @@ public void close() throws SQLException {
         return dataSets;
     }
 
+    boolean insert(String tableName, DataSet dataset) {
+        String dataSetFormatted = dataSetFormatted(dataset);
+        String sqlQuery = "INSERT INTO " + tableName + " VALUES(" + dataSetFormatted + ")";
+        try (Statement statement = connection.createStatement()) {
+            statement.executeUpdate(sqlQuery);
+            return true;
+        } catch (SQLException e) {
+            return false;
+        }
+    }
+
+
+    private String dataSetFormatted(DataSet dataSet) {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (String value : dataSet.values()) {
+            stringBuilder.append("'")
```

Comment:
Maybe use local variable such as wrapperSymbol or smth?

Reply:
Yeah, I Thought about this. But it will overload the code, I think. 
Ok, I'll try to use it.